app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result_list = [Person(person["name"], person["age"]) for person in people]
+
+    for num, person in enumerate(people):
+        if "wife" in person and person["wife"] is not None:
+            result_list[num].wife = Person.people[person["wife"]]
+        if "husband" in person and person["husband"] is not None:
```

Comment:
You can simplify the check by using the .get() dictionary method. If the dictionary does not have such a key, it will return None.